### PR TITLE
👻 Remove gas settings before sign step

### PR DESCRIPTION
**Pull request by henryboldi**
Swap
Send

⚠️ Would like to add an explanation behind this choice to the PR description, but feel free to review now if we need it quickly

### CODE REVIEW

File: ui/pages/Send.tsx [hunk 1]
```diff
@@ -71,17 +63,14 @@ export default function Send(): ReactElement {
     string | undefined
   >(undefined)
   const [amount, setAmount] = useState("")
-  const [gasLimit, setGasLimit] = useState<bigint | undefined>(undefined)
+  const [gasLimit] = useState<bigint | undefined>(undefined)
```
Comment (0xDaedalus) [inline ui/pages/Send.tsx:66]: I think we can remove this - its just undefined forever right?

File: ui/_locales/en/messages.json [hunk 2]
```diff
@@ -25,8 +25,7 @@
     "sendAsset": "Send Asset",
     "assetAmount": "Asset / Amount",
     "sendTo": "Send To:",
-    "estimatedFee": "Estimated network fee",
-    "sendButton": "Send",
+    "sendButton": "Continue",
```
Comment (0xDaedalus) [inline ui/_locales/en/messages.json:28]: ❤️

File: ui/pages/Send.tsx (hunk 1 above)
Comment (henryboldi) [reply ui/pages/Send.tsx]: Done!

**Review by 0xDaedalus — APPROVED** [review]
Looks great - swaps work - rock & roll

**Pull request merged**